modules/apps/headless/headless-site/headless-site-test/src/testIntegration/java/com/liferay/headless/site/resource/v1_0/test/SiteResourceTest.java

```diff
@@ -212,13 +221,14 @@ public void testPostSite() throws Exception {
 		_testPostSiteWithoutAuthentication();
 	}
 
-	@LazyReferencing
```

Comment:
Do we lose anything by removing this? What happens with the `_testPutSiteBatch` now?

Reply:
I did this because this annotation enables the `LazyReference` for all the tests in `testPutSite`, but I wanted to test the scenarios where it is disabled, so I removed this and activated the `LazyReference` explicitly in `_testPutSiteBatchSiteParentSiteERC`.

Reply:
Do we need to do the same for `_testPutSiteBatch` then?

Reply:
Could you debug through this test and see if it's enabled or not? I'm not sure unfortunately, just trying to follow other examples.

Reply:
Oh sorry, I think I missunderstood your comment, I think we need to also do it for `_testPutSiteBatch`